.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: python
+python:
+  - "2.7"
+install:
+  - pip install ansible==2.2.2
+script:
+  - cd ansible && ansible-playbook --version
```

Comment:
Yep, that's possible, but it needs some vault variables listing method. Accurate `vault_` prefix manipulation and stripping the prefix in corresponding `vars.yml` should work :)